SecurityService.OpenIdConnect.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -5,104 +5,241 @@
+                                           RedirectStandardOutput = true,
+                                           CreateNoWindow = true
+                                       }
+                       };
+            Console.WriteLine(proc.StartInfo.Arguments);
 
-        public ISecurityServiceClient SecurityServiceClient;
-        
+            proc.Start();
+            proc.WaitForExit();
+
+            return proc.StandardOutput.ReadToEnd();
+        }
+
+        private static IContainerService SetupSecurityServiceContainer(String containerName,
+                                                                      ILogger logger,
+                                                                      String imageName,
+                                                                      INetworkService networkService,
+                                                                      String hostFolder,
+                                                                      Int32 dockerPort,
+                                                                      (String URL, String UserName, String Password)? dockerCredentials,
+                                                                      Boolean forceLatestImage = false,
+                                                                      List<String> additionalEnvironmentVariables = null)
+        {
+            logger.LogInformation("About to Start Security Container");
+
+            List<String> environmentVariables = new List<String>();
+            environmentVariables.Add($"ServiceOptions:PublicOrigin=http://{containerName}:{dockerPort}");
+            environmentVariables.Add($"ServiceOptions:IssuerUrl=http://{containerName}:{dockerPort}");
+            environmentVariables.Add("ASPNETCORE_ENVIRONMENT=IntegrationTest");
+            environmentVariables.Add($"urls=http://*:5551");
+
+            if (additionalEnvironmentVariables != null)
+            {
+                environmentVariables.AddRange(additionalEnvironmentVariables);
+            }
+
+            String containerFolder = FdOs.IsLinux() ? "/home/txnproc/trace" : "C:\\home\\txnproc\\trace";
+            ContainerBuilder securityServiceContainer = new Builder().UseContainer().WithName(containerName)
+                                                                     .WithEnvironment(environmentVariables.ToArray()).UseImage(imageName, forceLatestImage)
+                                                                     .ExposePort(dockerPort, 5551).UseNetwork(new List<INetworkService>
+                                                                                                            {
+                                                                                                                networkService
+                                                                                                            }.ToArray());//.Mount(hostFolder,containerFolder,MountType.ReadWrite);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6144581)